src/components/Book.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Book = ({ book, handleRemoveBook }) => {
+  const { id, title, category } = book;
+  return (
+    <tr>
+      <td>
+        {id}
+        {' '}
+      </td>
+      <td>
+        {title}
+        {' '}
+      </td>
+      <td>
+        {category}
+        {' '}
+      </td>
+      <td><button type="button" onClick={e => { e.preventDefault(); handleRemoveBook(id); }}>Remove</button></td>
```

Comment:
- [ ] This is not a form so no need for preventing Default. Kindly remove it also.